### ISSUE TITLE
Maps not propagating

### ISSUE THREAD

**Issue by Tam** [ethercreative/simplemap#141]
Maps don't propagate across sites.

**Comment by vandres** [comment]
Just as a comment: The Map field should respect "translation method"

**Comment by peteeveleigh** [comment]
Can confirm. The field will work on the default site but doesn't seem to work on entries for other sites in a multi site setup.

**Comment by Antimated** [comment]
Any updates on when this will be fixed? :)

**Comment by stefangallas** [comment]
For a new project I came across the same bug. I would look forward to a bugfix, too. On the other hand, I have implemented a temporary workaround by adding the SiteId, under which the entries were originally entered, for the affected entry query .

**Comment by mmikkel** [comment]
I started on a PR to fix this issue, but it proved too time-consuming. I did implement a working hotfix though – Maps are implemented as a custom Element type, so by listening to the `Elements::EVENT_AFTER_SAVE_ELEMENT` it's possible to "manually" propagate a Map being saved to all other supported sites.

[Here's the code](https://gist.github.com/mmikkel/23910270590aa560604ef3cf2eff0f50) – this should be added to a custom Craft module or plugin. I can't promise that it's airtight, but it's working well on my end, for the moment.

Note that the hotfix a) will always propagate the Map field value across sites for new elements, and b) does not account for all the different translation methods; simply whether the field is set to "Not translatable" (any changes to the field will also be propagated across sites) or not (the field will only be propagated when the element is first saved, not for any further changes to the field).
